app/assets/stylesheets/arclight/modules/search_results.scss

```diff
@@ -149,12 +149,28 @@ article.document {
   fill: $gray-600;
 }
 
-.btn-outline-secondary:not(:disabled):not(.disabled).active svg, .btn-outline-secondary:hover svg {fill: #fff;}
-
-.btn-outline-secondary:not(:disabled):not(.disabled).active, .show > .btn-outline-secondary.dropdown-toggle {
-    color: #fff;
-    fill: white;
-    background-color: #6c757d;
-    border-color: #6c757d;
+.btn-outline-secondary:not(:disabled):not(.disabled).active svg, .btn-outline-secondary:hover svg {
+  fill: #fff;
 }
 
+@media only screen and (max-width: 767px) {
```

Comment:
I think flexbox may be a more maintainable approach for implementing the responsive design. I believe @jkeck may have some  flexbox work in flight that might be good to look at.

Reply:
I'd be interested to see how he accomplished that. I think it would take a lot of reworking the markup. I'm not a fan of absolute positioning, but I wasn't able to get the order to change the way I needed it to.

Reply:
I would be happy to pair up on this tomorrow.  The work I did was to use flex box for positioning the title and the bookmark, online content label, etc.  I agree that doing this w/ flex box will require some changes to the layout (but we own the layout for these documents at this point, so hopefully that shouldn't be too onerous). 

This might be a good opportunity to clean-up some of the inconsistencies between the default and compact views that have evolved over time as well.

Reply:
That would be great to pair up on this.

Reply:
Sorry I wasn't able to get together this afternoon (meeting schedule got a bit out of hand).

Started playing w/ some [flex box solutions](https://github.com/projectblacklight/arclight/tree/weird-flex) here, and it seems doable, but it would be helpful to get a few answers from the designers about content wrapping and  margin/padding expectations between the title and actions/tools (which I believe are illustrated in the screenshots below)

## Small
<img width="547" alt="sm" src="https://user-images.githubusercontent.com/96776/65299266-a8922880-db23-11e9-95ac-a1c8d88090b4.png">

## Medium or greater
<img width="551" alt="md" src="https://user-images.githubusercontent.com/96776/65299265-a8922880-db23-11e9-9c6d-daecadc802d8.png">

Reply:
I'm confused by these two screenshots. The "medium" is the same width as the "small"; I can't see how the screen would ever be in this state. Unless I've mis-stated the breakpoint somewhere?

Reply:
Hmm, I believe that may just be the screenshots being close in size (I may have taken the screenshots on very similar widths on either side of the breakpoint).

In my screenshots I'm using the Bootstrap medium breakpoint (not defining my own) so the small breakpoint (or smaller) gets the actions on the bottom and the medium breakpoint (or larger) gets the actions aligned with the title.

Reply:
I _think_ (based on looking at this PR locally) the screenshot confusion is due to differences in the way the "Small" and "Medium or greater" screenshots were taken. The "Small" version is showing the entire width of the viewport (at small, the content area takes full-width of the viewport/content area). The "Medium or greater" screenshot is only showing the 9 columns used for the content area; the is a 3-column sidebar (with the facets) not included in the screenshot. So the displayed content area is very similar in width between the two screenshot examples but the "Medium" actually includes more content to the left that is not shown.

That aside, the "Medium or greater" layout isn't great, especially with the way the container info wraps to three lines. However, I believe that issue will go away if we follow through with the [Blacklight ticket recommending we change the breakpoint](https://github.com/projectblacklight/blacklight/issues/2133) at which the sidebar switches from full-width to 3-cols. No one has objected to that, so maybe I (or someone else on the ArcLight team) can submit a PR for it this week. After we do that, the "Medium or greater" breakpoint should look more like this:

<img width="851" alt="Screen Shot 2019-09-23 at 8 56 35 AM" src="https://user-images.githubusercontent.com/101482/65442595-4ebe8680-dde1-11e9-8728-da90205665a4.png">

When we get to the next breakpoint, Large, the sidebar would be shown as 3 columns, but the additional width of the viewport at that breakpoint provides room for roughly the same layout of the search result items as the screenshot above.

Reply:
I merged in @jkeck work into this PR. We will need to verify the layout on medium screens looks ok with the additional space after Blacklight ticket is completed